### PR TITLE
Adding twitter demo

### PR DESCRIPTION
**Pull request by dmehra**
connects to #15 

Encountered issue https://github.com/juttle/juttle-twitter-adapter/issues/10

@demmer

### CODE REVIEW

File: demos/twitter-race/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+The user inputs 2 search terms, juttle parses the incoming tweets to find matches, and displays match count as live-updating tiles and a timechart,
+to visually compare popularity of the requested terms.
+
+## Setup
+
+We will need outrigger to render visualizations in the browser, and the twitter adapter to enable reading. 
+
+```
+npm install -g juttle
+npm install -g outrigger
```
Comment (demmer) [inline demos/twitter-race/README.md]: Once https://github.com/juttle/outrigger/issues/48 is fixed we won't need to explicitly install juttle or the adapter. I think it makes sense to future proof these instructions and just tell people to install outrigger and then configure the adapter.

**Comment by mstemm** [comment]
I made additional changes to run using docker and the outrigger image now available on docker hub. This should be ready to use.

@dmehra @demmer

File: demos/twitter-race/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+To obtain these credentials, set up a [Twitter App](https://apps.twitter.com/) using your twitter username, and create an OAuth token.
+
+### Starting outrigger
+
+Start outrigger via docker:
+
+```
+docker run --name outrigger -p 8080:8080 -v `pwd`/juttles:/opt/outrigger/juttles -v `pwd`/juttle-config.json:/opt/outrigger/.juttle-config.json juttle/outrigger:latest
+```
```
Comment (mstemm) [inline demos/twitter-race/README.md]: Daria pointed out that someone might have another copy of outrigger-in-a-container running. In that case, we should add a note here "If you're running another copy of outrigger, change the -p argument to something like -p 9090:8080 to access the second outrigger instance.

Comment (dmehra) [inline demos/twitter-race/README.md]: Add a note that if you're already running a docker outrigger demo, you should change the command to `--name outrigger1 -p 8081:8080` or similar, i.e. change both the container name, and the local port.

**Comment by dmehra** [comment]
please merge

**Pull request merged**